hw/ip_templates/ac_range_check/data/ac_range_check.hjson.tpl

```diff
@@ -179,7 +179,9 @@ import math
     { name: "LOG_CONFIG"
       desc: ""
       swaccess: "rw"
-      hwaccess: "hro"
+      hwaccess: "hrw"
```

Comment:
The `log_enable` and `deny_cnt_threshold` fields are still read-only by HW, right?

Reply:
They are RO in hardware. But because `log_config` is `hwext`and SW-RW, they also need to be hardware RW. If you would change the subfield to `hwaccess: "hro"`, you would get:
```
ValueError: The LOG_CONFIG register has hwext set, but field deny_cnt_threshold has hro hwaccess and the field value is readable by software (mode rw).
```